dns/route53/dns_test.go

```diff
@@ -2,79 +2,117 @@ package route53
 
 import (
 	"testing"
+	"time"
 
+	"fmt"
 	"net"
 
 	"github.com/aws/aws-sdk-go/aws"
-	route53Client "github.com/aws/aws-sdk-go/service/route53"
+	"github.com/aws/aws-sdk-go/service/route53"
 	log "github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestEditResourceRecords(t *testing.T) {
+const (
+	MOCKED_ZONE_ID = "20N31D"
+)
+
+type mockR53 struct {
+	recordSets map[string]*route53.ResourceRecordSet
+}
+
+func (r53 *mockR53) ChangeResourceRecordSets(input *route53.ChangeResourceRecordSetsInput) (*route53.ChangeResourceRecordSetsOutput, error) {
```

Comment:
While not strictly necessary, a line describing what this does shouldn't really hurt.

Reply:
IMHO, the code and the function name is sufficiently self-explanatory.